truss/remote/baseten/api.py

```diff
@@ -157,23 +157,22 @@ def models(self):
     def get_model(self, model_name):
```

Comment:
The changes to `get_model` should be safe--the current callsites are https://github.com/basetenlabs/truss/blob/8901a0121dda36b822fbabe62f8a172ba72c6c6e/truss/remote/baseten/core.py#L50C1-L61 and are updated in this PR